Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -149,75 +151,47 @@ public void PullRequestCacheBasePath_EmptyDirectories_IsNull(string sourcesDirec
-        }
-
-        [TestMethod]
-        public async Task Execute_PullRequest_SonarQubePrior99()
-        {
-            using var sut = new CacheProcessor(Mock.Of<ISonarWebService>(x => x.ServerVersion == SonarQubeVersion98), CreateProcessedArgs("/k:key /d:sonar.pullrequest.base=master"), Mock.Of<IBuildSettings>(), logger);
-            await sut.Execute();
-
-            logger.AssertDebugLogged("Incremental PR analysis is available starting with SonarQube 9.9 or later.");
-            sut.UnchangedFilesPath.Should().BeNull();
-        }
-
         [TestMethod]
         public async Task Execute_PullRequest_NoBasePath()
         {
-            using var sut = new CacheProcessor(MockSonarWebService(), CreateProcessedArgs("/k:key /d:sonar.pullrequest.base=master"), Mock.Of<IBuildSettings>(), logger);
+            using var sut = new CacheProcessor(server, CreateProcessedArgs("/k:key /d:sonar.pullrequest.base=master"), Mock.Of<IBuildSettings>(), logger);
+
             await sut.Execute();
 
-            logger.AssertWarningLogged("Cannot determine project base path. Incremental PR analysis is disabled.");
+            logger.AssertSingleInfoMessageExists("Cannot determine project base path. Incremental PR analysis is disabled.");
             sut.UnchangedFilesPath.Should().BeNull();
         }
 
         [TestMethod]
-        public async Task Execute_PullRequest_NoCache()
+        public async Task Execute_PullRequest_CacheNull()
         {
             var settings = Mock.Of<IBuildSettings>(x => x.SourcesDirectory == @"C:\Sources");
-            using var sut = new CacheProcessor(MockSonarWebService(), CreateProcessedArgs("/k:key /d:sonar.pullrequest.base=TARGET_BRANCH"), settings, logger);
+            using var sut = new CacheProcessor(server, CreateProcessedArgs("/k:key /d:sonar.pullrequest.base=TARGET_BRANCH"), settings, logger);
+
             await sut.Execute();
 
-            logger.AssertInfoLogged("Processing pull request with base branch 'TARGET_BRANCH'.");
+            logger.AssertInfoLogged("Cache data is not available. Incremental PR analysis is disabled.");
             sut.UnchangedFilesPath.Should().BeNull();
         }
 
         [TestMethod]
-        public async Task Execute_PullRequest_FullProcessing_NoCacheAvailable()
+        public async Task Execute_PullRequest_CacheEmpty()
         {
             var context = new CacheContext(this, "/k:key-no-cache /d:sonar.pullrequest.base=TARGET_BRANCH");
+
             await context.Sut.Execute();
 
-            logger.AssertInfoLogged("Processing pull request with base branch 'TARGET_BRANCH'.");
             logger.AssertInfoLogged("Cache data is not available. Incremental PR analysis is disabled.");
             context.Sut.UnchangedFilesPath.Should().BeNull();
         }
 
         [TestMethod]
-        public async Task Execute_PullRequest_FullProcessing_WithCache()
+        public async Task Execute_PullRequest_CacheFull()
```

Comment:
I would rename this to `Execute_PullRequest_CacheHappyFlow` since `full` is not clear here